packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -112,8 +99,17 @@ const TradersHub = () => {
                     {scrolled && <TourGuide />}
                 </div>
             </Div100vhContainer>
-            {is_eu_low_risk && <EUDisclamer />}
-        </>
+            {is_eu_low_risk && (
+                <div className='disclamer'>
```

Comment:
Conditionally rendered the text directly as it was recreating the outside component during each render.

Reply:
this react issue is fixed in this PR https://github.com/binary-com/deriv-app/pull/9339/files
looks like a merge conflict is coming in for me.. 😄

Reply:
brace yourself @shaheer-deriv 😄